Game/Game.cls

```diff
@@ -35,6 +35,7 @@ public with sharing class Game
 
 		fakeVar = fakeVar;
 		blah = fakeVar;
+		blah = blah;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Avoid idempotent operations (like assigning a variable to itself). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://app.codescan.io/coding_rules#rule_key=sf%3AIdempotentOperations)